기본 url 접속시 page list 보여주기

bsh714-dev opened the pull request:
#27

bsh714-dev committed:
reset

bsh714-dev committed:
On branch feat_bsh
 #27 기본 url 접속시 page list 보여주기

 Changes to be committed:
	new file:   .vscode/extensions.json
	modified:   package-lock.json
	modified:   package.json
	modified:   src/app.js
	modified:   src/http/http.controller.js
	modified:   src/http/http.repository.js
	modified:   src/http/http.service.js
	modified:   src/routes/http.route.js
	modified:   src/routes/index.route.js
	modified:   src/schemas/index.schema.js
	modified:   yarn.lock

bsh714-dev committed:
On branch bsh714
 #27 기본 url 접속시 page list 보여주기
 Changes to be committed:
	modified:   src/http/http.service.js

jn33-dev committed:
Merge branch 'dev' into feat_bsh

bsh714-dev committed:
페이지가 없을경우 예외처리
 Changes to be committed:
	modified:   src/http/http.controller.js

bsh714-dev committed:
Merge branch 'feat_bsh' of https://github.com/bla-bla-Notion/backend into bsh714

jn33-dev committed:
Delete extensions.json

vs code extension 파일 삭제

jn33-dev commented on src/schemas/index.schema.js:
```diff
@@ -14,47 +14,45 @@ const connect = async () => {
-  });
-  console.log(result);
-})();
+// (async () => {
```
주석처리 된 부분 삭제 부탁드립니다~

jn33-dev commented on src/http/http.service.js:
```diff
@@ -1,5 +1,22 @@
+  }
+  findPageList = async () => {
+    const pageList = await this.httpRepository.findPage();
+    console.log(pageList);
```
콘솔 로그 삭제해주세요~

jn33-dev commented on src/http/http.controller.js:
```diff
@@ -1,11 +1,18 @@
 const HttpService = require('./http.service');
+//const {} = require('../exception/index.exception.js');
```
주석처리 된 부분 필요 없으니 삭제해주세요.

jn33-dev commented on src/http/http.service.js:
```diff
@@ -1,5 +1,22 @@
 const { redisClient } = require('../schemas/index.schema');
```
여기서 사용하지 않으니 삭제해주세요.

jn33-dev commented on src/http/http.service.js:
```diff
@@ -1,5 +1,22 @@
+  findPageList = async () => {
+    const pageList = await this.httpRepository.findPage();
+    console.log(pageList);
+    pageList.sort((a, b) => {
```
이미 배열이 생성된 순서로 정렬되어 있기 때문에, sort가 필요 없습니다.

jn33-dev reviewed (changes requested):
주석처리, console.log 등 삭제해주세요!

bsh714-dev committed:
Merge branch 'feat_bsh' of https://github.com/bla-bla-Notion/backend into bsh714

bsh714-dev committed:
FIX: 주석제거 콘솔로그제거 등

 Changes to be committed:
	modified:   src/http/http.controller.js
	modified:   src/http/http.service.js
	modified:   src/schemas/index.schema.js
